recipe/meta.yaml

```diff
@@ -51,24 +50,22 @@ requirements:
     - pybind11 <2.11
     - pcre
     - gflags
-    - cyrus-sasl =2.1.27
     - msgpack-c
     - aws-sdk-cpp
     - azure-core-cpp
     - azure-identity-cpp
     - azure-storage-blobs-cpp
     - bitmagic
-    - krb5 =1.20.1
     - xxhash
     - semimap
     - grpcio-tools
     - protobuf
     - spdlog
     - robin_hood
     - recycle
-    - lmdb ==0.9.22
-    - lmdbxx ==0.9.14
-    - fmt <10
```

Comment:
I think you can keep the `fmt` constraint here

Reply:
Done in 8be20ec.